### DISCUSSION TITLE
How do I check the file date against the date entry in the layout.json file

### DISCUSSION THREAD

**Discussion by dinther** [mpaperno/WASimCommander#20] [General]
I load the layout.json file using Newtonsoft.Json into an object that keeps the Date as a long
Then  I obtain the date value from the file like so:

```
FileInfo fileInfo = new FileInfo(filePath);
if (fileInfo.LastWriteTimeUtc.ToFileTimeUtc() == item.date) ... 
```

For `Thubnail.jpg` I get `132989077060000000` rather than `132989725046757824` from the layout.json file.
I am unable to get a match. I started to wonder if the layout.json file is maybe  wrong so I generated a layout.json file with [MSFSLayoutGenerator](https://github.com/HughesMDflyer4/MSFSLayoutGenerator)

The layout.json file generated contains the number `132989077060000000` which would match the number I get.
Am I doing something wrong or is the layout.json file not up to date?

**Comment by mpaperno** [comment]
The module package is built with MSFS (command-line mode). It works, which is about as far as I've looked into layout.json details.

The timestamp you say is in the current layout.json, `132989725046757824`, is correct (when decoded it matches exactly the last mod time of the thumbnail file). The other number you quote is not (almost 24 hours off).  https://www.epochconverter.com/ldap

-Max

![image](https://github.com/mpaperno/WASimCommander/assets/1366615/355b77ee-94f0-47ec-9b18-ba138173a467)

(Local time is UTC-4)
